api/operator/v1alpha1/istiocsr_types.go

```diff
@@ -146,6 +146,30 @@ type CertManagerConfig struct {
 	// +kubebuilder:validation:Required
 	// +required
 	IssuerRef certmanagerv1.ObjectReference `json:"issuerRef,omitempty"`
+
+	// istioCACertificate when provided, the operator will use the CA certificate from the specified ConfigMap. If empty, the operator will
+	// automatically extract the CA certificate from the Secret containing the istiod certificate obtained from cert-manager.
+	// +kubebuilder:validation:Optional
+	// +optional
+	IstioCACertificate *ConfigMapReference `json:"istioCACertificate,omitempty"`
+}
+
+// ConfigMapReference holds the details of a configmap.
+type ConfigMapReference struct {
```

Comment:
I will update the API as per the EP conclusion: https://github.com/openshift/enhancements/pull/1837